veggieseasons/lib/styles.dart

```diff
@@ -116,4 +118,10 @@ abstract class Styles {
     fontFamily: CupertinoIcons.iconFont,
     fontPackage: CupertinoIcons.iconFontPackage,
   );
+
+  static const shadowGradient = LinearGradient(
+    begin: Alignment.topCenter,
+    end: Alignment.bottomCenter,
+    colors: [Color(0x00000000), Color(0xa0000000)],
```

Comment:
nit: consider pulling this color out and making it a constant like some of the ones above. Or disregard. I'm a code reviewer with vague opinions, not a cop. ;-)

Reply:
Done. 😄